Bug: Force NPM symbolic links for Grunt

While helping someone on the VVV slack we encountered this issue:

<details>

```
\ V\ V\ V / Varying Vagrant Vagrants
 \_/\_/\_/  v3.7.0-git::develop(35b54c0)

Platform: platform-linux-gnu shell:/bin/bash systemd  vagrant-goodhosts vagrant-vbguest CaseSensitiveFS shared_db_folder_disabled, VVV Path: "/home/myusername/vvv"
Vagrant: v2.2.10, VirtualBox: v6.1.18r142142

Docs:       https://varyingvagrantvagrants.org/
Contribute: https://github.com/varying-vagrant-vagrants/vvv
Dashboard:  http://vvv.test

Bringing machine 'default' up with 'virtualbox' provider...
==> default: Checking if box 'ubuntu/bionic64' version '20210319.0.0' is up to date...
==> default: A newer version of the box 'ubuntu/bionic64' for provider 'virtualbox' is
==> default: available! You currently have version '20210319.0.0'. The latest is version
==> default: '20210521.0.0'. Run `vagrant box update` to update.
==> default: Clearing any previously set forwarded ports...
==> default: Clearing any previously set network interfaces...
==> default: Preparing network interfaces based on configuration...
    default: Adapter 1: nat
    default: Adapter 2: hostonly
==> default: Forwarding ports...
    default: 22 (guest) => 2222 (host) (adapter 1)
==> default: Running 'pre-boot' VM customizations...
==> default: Booting VM...
==> default: Waiting for machine to boot. This may take a few minutes...
    default: SSH address: 127.0.0.1:2222
    default: SSH username: vagrant
    default: SSH auth method: private key
    default: Warning: Connection reset. Retrying...
    default: Warning: Remote connection disconnect. Retrying...
==> default: Machine booted and ready!
==> default: Checking for guest additions in VM...
    default: The guest additions on this VM do not match the installed version of
    default: VirtualBox! In most cases this is fine, but in rare cases it can
    default: prevent things such as shared folders from working properly. If you see
    default: shared folder errors, please make sure the guest additions within the
    default: virtual machine match the version of VirtualBox you have installed on
    default: your host and reload your VM.
    default: 
    default: Guest Additions Version: 5.2.8_KernelUbuntu r120774
    default: VirtualBox Version: 6.1
==> default: [vagrant-goodhosts] Checking for host entries
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 vvv.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 local.wordpress.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 fse.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 clone.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 zach.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 book.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 plain.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 case.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 tideways.vvv.test
==> default: [vagrant-goodhosts] - found entry for: 192.168.50.4 xhgui.vvv.test
==> default: Setting hostname...
==> default: Configuring and enabling network interfaces...
==> default: Mounting shared folders...
    default: /srv/www => /home/myusername/vvv/www
    default: /srv/config => /home/myusername/vvv/config
    default: /var/log/php => /home/myusername/vvv/log/php
    default: /srv/database => /home/myusername/vvv/database/sql
    default: /srv/provision => /home/myusername/vvv/provision
    default: /var/log/nginx => /home/myusername/vvv/log/nginx
    default: /srv/certificates => /home/myusername/vvv/certificates
    default: /var/log/memcached => /home/myusername/vvv/log/memcached
    default: /var/log/provisioners => /home/myusername/vvv/log/provisioners
==> default: Detected mount owner ID within mount options. (uid: 0 guestpath: /var/log/memcached)
==> default: Detected mount group ID within mount options. (gid: 106 guestpath: /var/log/memcached)
==> default: Detected mount owner ID within mount options. (uid: 0 guestpath: /var/log/nginx)
==> default: Detected mount group ID within mount options. (gid: 106 guestpath: /var/log/nginx)
==> default: Detected mount owner ID within mount options. (uid: 0 guestpath: /var/log/php)
==> default: Detected mount group ID within mount options. (gid: 106 guestpath: /var/log/php)
==> default: Detected mount owner ID within mount options. (uid: 0 guestpath: /var/log/provisioners)
==> default: Detected mount group ID within mount options. (gid: 106 guestpath: /var/log/provisioners)
==> default: Detected mount owner ID within mount options. (uid: 1000 guestpath: /srv/www)
==> default: Detected mount group ID within mount options. (gid: 33 guestpath: /srv/www)
==> default: Running provisioner: file...
    default: /home/myusername/vvv/version => /home/vagrant/version
==> default: Running provisioner: pre-provision-script (shell)...
    default: Running: inline script
    default: 
    default: ༼ つ ◕_◕ ༽つ A full provision can take a little while!
    default:              Go make a cup of tea and sit back.
    default:              If you only wanted to turn VVV on, use vagrant up
==> default: Running provisioner: default (shell)...
    default: Running: /tmp/vagrant-shell20210525-13167-1362xyg.sh
    default:  ▷ Running the 'main' provisioner...
    default:  ▷ Running init hook
    default:  * Bash profile setup and directories.
    default:  * Setting ownership of files in /home/vagrant to vagrant
    default:  * Copying /srv/config/bash_profile                      to /home/vagrant/.bash_profile
    default:  * Copying /srv/config/bash_aliases                      to /home/vagrant/.bash_aliases
    default:  * Copying /srv/config/bash_aliases                      to /root/.bash_aliases
    default:  * Copying /srv/config/vimrc                             to /home/vagrant/.vimrc
    default:  * Copying /srv/config/subversion-servers                to /home/vagrant/.subversion/servers
    default:  * Copying /srv/config/subversion-config                 to /home/vagrant/.subversion/config
    default:  * Copying /srv/config/ssh_known_hosts                   to /etc/ssh/ssh_known_hosts
    default:  * Copying /srv/config/sshd_config                       to /etc/ssh/sshd_config
    default:  * Reloading SSH Daemon
    default:  * checking Ubuntu version
    default:  ✔ Finished init hook in 2s
    default:  * Testing network connection to http://ppa.launchpad.net
    default:  * Successful Network connection to http://ppa.launchpad.net detected
    default:  * Testing network connection to https://wordpress.org
    default:  * Successful Network connection to https://wordpress.org detected
    default:  * Testing network connection to https://github.com
    default:  * Successful Network connection to https://github.com detected
    default:  * Testing network connection to https://raw.githubusercontent.com
    default:  * Successful Network connection to https://raw.githubusercontent.com detected
    default:  * Testing network connection to https://getcomposer.org
    default:  * Successful Network connection to https://getcomposer.org detected
    default:  * Testing network connection to http://ams2.mirrors.digitalocean.com
    default:  * Successful Network connection to http://ams2.mirrors.digitalocean.com detected
    default:  * Testing network connection to https://deb.nodesource.com
    default:  * Successful Network connection to https://deb.nodesource.com detected
    default:  * Network checks succeeded
    default:  ▷ Running before_packages hook
    default:  * git-core/ppa already present, skipping
    default:  * Setting up MySQL configuration file links...
    default:  * mysql group exists
    default:  * mysql user present and has uid 9001
    default:  * Copying /srv/config/mysql-config/vvv-core.cnf to /etc/mysql/conf.d/vvv-core.cnf
    default:  * Copying PHP configs
    default:  * Checking supplementary PHP configs
    default:  ✔ Finished before_packages hook in 3s
    default:  * Main packages check and install.
    default:  * Cleaning up dpkg lock file
    default:  * Updating apt keys
    default: gpg: key 3B4FE6ACC0B21F32: 3 signatures not checked due to missing keys
    default: gpg: key 3B4FE6ACC0B21F32: "Ubuntu Archive Automatic Signing Key (2012) <ftpmaster@ubuntu.com>" not changed
    default: gpg: key D94AA3F0EFE21092: 3 signatures not checked due to missing keys
    default: gpg: key D94AA3F0EFE21092: "Ubuntu CD Image Automatic Signing Key (2012) <cdimage@ubuntu.com>" not changed
    default: gpg: key 871920D1991BC93C: 1 signature not checked due to a missing key
    default: gpg: key 871920D1991BC93C: "Ubuntu Archive Automatic Signing Key (2018) <ftpmaster@ubuntu.com>" not changed
    default: gpg: Total number processed: 3
    default: gpg:              unchanged: 3
    default:  * Running apt-get update...
    default: Ign:1 https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 InRelease
    default: Get:2 http://ams2.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic InRelease [6265 B]
    default: Get:3 https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 Release [2989 B]
    default: Get:4 https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0 Release.gpg [801 B]
    default: Get:5 http://ams2.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main Sources [1965 B]
    default: Get:6 https://repo.mongodb.org/apt/ubuntu bionic/mongodb-org/4.0/multiverse amd64 Packages [15.9 kB]
    default: Get:7 http://ams2.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main amd64 Packages [17.3 kB]
    default: Get:8 https://nginx.org/packages/mainline/ubuntu bionic InRelease [2862 B]
    default: Get:9 https://deb.nodesource.com/node_14.x bionic InRelease [4584 B]
    default: Get:10 http://ams2.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main ppc64el Packages [16.6 kB]
    default: Get:11 http://security.ubuntu.com/ubuntu bionic-security InRelease [88.7 kB]
    default: Get:12 http://ams2.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main arm64 Packages [16.6 kB]
    default: Get:13 http://ftp.yz.yamagata-u.ac.jp/pub/dbms/mariadb/repo/10.5/ubuntu bionic InRelease [6265 B]
    default: Get:15 http://archive.ubuntu.com/ubuntu bionic InRelease [242 kB]
    default: Get:16 http://mirrors.ukfast.co.uk/sites/mariadb/repo/10.5/ubuntu bionic InRelease [6265 B]
    default: Get:17 http://sfo1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic InRelease [6265 B]
    default: Get:18 https://nginx.org/packages/mainline/ubuntu bionic/nginx Sources [70.6 kB]
    default: Get:19 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages [1726 kB]
    default: Get:20 http://sgp1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic InRelease [6265 B]
    default: Get:21 http://ftp.yz.yamagata-u.ac.jp/pub/dbms/mariadb/repo/10.5/ubuntu bionic/main Sources [1965 B]
    default: Get:22 http://ppa.launchpad.net/git-core/ppa/ubuntu bionic InRelease [20.8 kB]
    default: Get:23 http://mirrors.ukfast.co.uk/sites/mariadb/repo/10.5/ubuntu bionic/main Sources [1965 B]
    default: Get:24 http://sfo1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main Sources [1965 B]
    default: Get:25 http://ftp.yz.yamagata-u.ac.jp/pub/dbms/mariadb/repo/10.5/ubuntu bionic/main amd64 Packages [17.3 kB]
    default: Get:26 https://deb.nodesource.com/node_14.x bionic/main amd64 Packages [765 B]
    default: Get:27 http://sgp1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main Sources [1965 B]
    default: Get:28 https://nginx.org/packages/mainline/ubuntu bionic/nginx amd64 Packages [71.3 kB]
    default: Get:29 http://sfo1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main ppc64el Packages [16.6 kB]
    default: Get:30 http://mirrors.ukfast.co.uk/sites/mariadb/repo/10.5/ubuntu bionic/main arm64 Packages [16.6 kB]
    default: Get:14 https://packagecloud.io/github/git-lfs/ubuntu bionic InRelease [23.2 kB]
    default: Get:31 http://security.ubuntu.com/ubuntu bionic-security/main Translation-en [322 kB]
    default: Get:32 http://sgp1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main arm64 Packages [16.6 kB]
    default: Get:33 http://sfo1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main amd64 Packages [17.3 kB]
    default: Get:34 http://archive.ubuntu.com/ubuntu bionic-updates InRelease [88.7 kB]
    default: Get:35 http://security.ubuntu.com/ubuntu bionic-security/restricted amd64 Packages [323 kB]
    default: Get:36 http://ftp.yz.yamagata-u.ac.jp/pub/dbms/mariadb/repo/10.5/ubuntu bionic/main ppc64el Packages [16.6 kB]
    default: Get:37 http://sfo1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main arm64 Packages [16.6 kB]
    default: Get:38 http://sgp1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main ppc64el Packages [16.6 kB]
    default: Get:39 http://ppa.launchpad.net/ondrej/php/ubuntu bionic InRelease [20.8 kB]
    default: Get:40 http://mirrors.ukfast.co.uk/sites/mariadb/repo/10.5/ubuntu bionic/main amd64 Packages [17.3 kB]
    default: Get:41 http://security.ubuntu.com/ubuntu bionic-security/restricted Translation-en [43.2 kB]
    default: Get:42 http://sgp1.mirrors.digitalocean.com/mariadb/repo/10.5/ubuntu bionic/main amd64 Packages [17.3 kB]
    default: Get:43 http://security.ubuntu.com/ubuntu bionic-security/universe amd64 Packages [1126 kB]
    default: Get:44 http://archive.ubuntu.com/ubuntu bionic-backports InRelease [74.6 kB]
    default: Get:45 http://mirrors.ukfast.co.uk/sites/mariadb/repo/10.5/ubuntu bionic/main ppc64el Packages [16.6 kB]
    default: Get:46 http://ftp.yz.yamagata-u.ac.jp/pub/dbms/mariadb/repo/10.5/ubuntu bionic/main arm64 Packages [16.6 kB]
    default: Get:47 http://security.ubuntu.com/ubuntu bionic-security/universe Translation-en [254 kB]
    default: Get:49 http://archive.ubuntu.com/ubuntu bionic/main amd64 Packages [1019 kB]
    default: Get:50 http://security.ubuntu.com/ubuntu bionic-security/multiverse amd64 Packages [19.2 kB]
    default: Get:51 http://security.ubuntu.com/ubuntu bionic-security/multiverse Translation-en [4412 B]
    default: Get:48 https://packagecloud.io/github/git-lfs/ubuntu bionic/main amd64 Packages [4387 B]
    default: Get:52 http://archive.ubuntu.com/ubuntu bionic/main Translation-en [516 kB]
    default: Get:53 http://ppa.launchpad.net/varying-vagrant-vagrants/php/ubuntu bionic InRelease [15.9 kB]
    default: Get:54 http://archive.ubuntu.com/ubuntu bionic/restricted amd64 Packages [9184 B]
    default: Get:55 http://archive.ubuntu.com/ubuntu bionic/restricted Translation-en [3584 B]
    default: Get:56 http://archive.ubuntu.com/ubuntu bionic/universe amd64 Packages [8570 kB]
    default: Get:57 http://ppa.launchpad.net/git-core/ppa/ubuntu bionic/main amd64 Packages [3044 B]
    default: Get:58 http://ppa.launchpad.net/git-core/ppa/ubuntu bionic/main Translation-en [2252 B]
    default: Get:59 http://ppa.launchpad.net/ondrej/php/ubuntu bionic/main Sources [32.6 kB]
    default: Get:60 http://ppa.launchpad.net/ondrej/php/ubuntu bionic/main amd64 Packages [97.0 kB]
    default: Get:61 http://archive.ubuntu.com/ubuntu bionic/universe Translation-en [4941 kB]
    default: Get:62 http://ppa.launchpad.net/ondrej/php/ubuntu bionic/main Translation-en [34.6 kB]
    default: Get:63 http://ppa.launchpad.net/varying-vagrant-vagrants/php/ubuntu bionic/main Sources [30.1 kB]
    default: Get:64 http://archive.ubuntu.com/ubuntu bionic/multiverse amd64 Packages [151 kB]
    default: Get:65 http://archive.ubuntu.com/ubuntu bionic/multiverse Translation-en [108 kB]
    default: Get:66 http://archive.ubuntu.com/ubuntu bionic-updates/main amd64 Packages [2069 kB]
    default: Get:67 http://ppa.launchpad.net/varying-vagrant-vagrants/php/ubuntu bionic/main amd64 Packages [92.4 kB]
    default: Get:68 http://archive.ubuntu.com/ubuntu bionic-updates/main Translation-en [413 kB]
    default: Get:69 http://ppa.launchpad.net/varying-vagrant-vagrants/php/ubuntu bionic/main Translation-en [18.5 kB]
    default: Get:70 http://archive.ubuntu.com/ubuntu bionic-updates/restricted amd64 Packages [344 kB]
    default: Get:71 http://archive.ubuntu.com/ubuntu bionic-updates/restricted Translation-en [46.8 kB]
    default: Get:72 http://archive.ubuntu.com/ubuntu bionic-updates/universe amd64 Packages [1735 kB]
    default: Get:73 http://archive.ubuntu.com/ubuntu bionic-updates/universe Translation-en [369 kB]
    default: Get:74 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse amd64 Packages [25.0 kB]
    default: Get:75 http://archive.ubuntu.com/ubuntu bionic-updates/multiverse Translation-en [6464 B]
    default: Get:76 http://archive.ubuntu.com/ubuntu bionic-backports/main amd64 Packages [10.0 kB]
    default: Get:77 http://archive.ubuntu.com/ubuntu bionic-backports/main Translation-en [4764 B]
    default: Get:78 http://archive.ubuntu.com/ubuntu bionic-backports/universe amd64 Packages [10.3 kB]
    default: Get:79 http://archive.ubuntu.com/ubuntu bionic-backports/universe Translation-en [4588 B]
    default: Fetched 25.5 MB in 8s (3120 kB/s)
    default: Reading package lists...
    default:  * Installing apt-get packages...
    default: Reading package lists...
    default: Building dependency tree...
    default: 
    default: Reading state information...
    default: make is already the newest version (4.1-9.1ubuntu1).
    default: python-setuptools is already the newest version (39.0.1-2).
    default: zip is already the newest version (3.0-11build1).
    default: colordiff is already the newest version (1.0.18-1).
    default: dos2unix is already the newest version (7.3.4-3).
    default: ngrep is already the newest version (1.47+ds1-1).
    default: subversion is already the newest version (1.9.7-4ubuntu1).
    default: curl is already the newest version (7.58.0-2ubuntu3.13).
    default: g++ is already the newest version (4:7.4.0-1ubuntu2.3).
    default: gettext is already the newest version (0.19.8.1-6ubuntu0.3).
    default: imagemagick is already the newest version (8:6.9.7.4+dfsg-16ubuntu6.9).
    default: lftp is already the newest version (4.8.1-1ubuntu0.2).
    default: memcached is already the newest version (1.5.6-0ubuntu1.2).
    default: postfix is already the newest version (3.3.0-1ubuntu0.3).
    default: software-properties-common is already the newest version (0.96.24.32.14).
    default: unzip is already the newest version (6.0-21ubuntu1.1).
    default: vim is already the newest version (2:8.0.1453-1ubuntu1.4).
    default: ntp is already the newest version (1:4.2.8p10+dfsg-5ubuntu7.3).
    default: ntpdate is already the newest version (1:4.2.8p10+dfsg-5ubuntu7.3).
    default: python-pip is already the newest version (9.0.1-2.3~ubuntu1.18.04.4).
    default: git is already the newest version (1:2.31.1-0ppa1~ubuntu18.04.1).
    default: git-svn is already the newest version (1:2.31.1-0ppa1~ubuntu18.04.1).
    default: git-lfs is already the newest version (2.13.3).
    default: mariadb-server is already the newest version (1:10.5.10+maria~bionic).
    default: nginx is already the newest version (1.21.0-1~bionic).
    default: nodejs is already the newest version (14.17.0-1nodesource1).
    default: php-pear is already the newest version (1:1.10.12+submodules+notgz+20210212-1+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-bcmath is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-cli is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-common is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-curl is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-dev is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-fpm is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-gd is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-imagick is already the newest version (3.4.4+php8.0+3.4.4-7+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-imap is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-intl is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-json is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-mbstring is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-memcache is already the newest version (8.0+4.0.5.2+3.0.9~20170802.e702b5f9+-1+ubuntu18.04.1+deb.sury.org+2).
    default: php7.4-memcached is already the newest version (3.1.5+2.2.0-9+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-mysql is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-pcov is already the newest version (1.0.8-1+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-soap is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-ssh2 is already the newest version (1.3.1+0.13-1+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-xdebug is already the newest version (3.0.3+2.9.8+2.8.1+2.5.5-1+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-xml is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-yaml is already the newest version (2.2.1+2.1.0+2.0.4+1.3.2-6+ubuntu18.04.1+deb.sury.org+1).
    default: php7.4-zip is already the newest version (7.4.18-2+ubuntu18.04.1+deb.sury.org+1).
    default: 0 upgraded, 0 newly installed, 0 to remove and 49 not upgraded.
    default:  * Removing unnecessary apt packages...
    default: Reading package lists...
    default: Building dependency tree...
    default: 
    default: Reading state information...
    default: 0 upgraded, 0 newly installed, 0 to remove and 49 not upgraded.
    default:  * Cleaning apt caches...
    default:  * Running tools_install
    default:  ▷ Running after_packages hook
    default:  * Turning off XDebug to avoid Composer performance issues
    default:  ✘ Disabling active PHP v8.0 debug mod: 'xhgui'
    default:  ✘ Disabling active PHP v7.4 debug mod: 'xhgui'
    default:  ✘ Disabling active PHP v7.3 debug mod: 'xhgui'
    default:  ✘ Disabling active PHP v8.0 debug mod: 'tideways_xhprof'
    default:  ✘ Disabling active PHP v7.4 debug mod: 'tideways_xhprof'
    default:  ✘ Disabling active PHP v7.3 debug mod: 'tideways_xhprof'
    default:  * Restarting PHP FPM services so that the change takes effect
    default:  ✔ All PHP Debug mods are now turned off.
    default:  * Making sure the composer cache is not owned by root
    default:  * Checking Composer is installed
    default:  * Checking for github tokens
    default:  * Checking for composer updates
    default:  * Updating Composer...
    default: Changed current directory to /usr/local/src/composer
    default: You are already using the latest available Composer version 2.0.14 (stable channel).
    default:  * Making sure the PHPUnit 7.5 package is available...
    default: Changed current directory to /usr/local/src/composer
    default: ./composer.json has been updated
    default:  * Updating global composer packages...
    default: Changed current directory to /usr/local/src/composer
    default: Loading composer repositories with package information
    default: Updating dependencies
    default: Nothing to modify in lock file
    default: Installing dependencies from lock file (including require-dev)
    default: Nothing to install, update or remove
    default: Package phpunit/php-token-stream is abandoned, you should avoid using it. No replacement was suggested.
    default: Generating autoload files
    default: 16 packages you are using are looking for funding.
    default: Use the `composer fund` command to find out more!
    default:  * Global composer package update completed
    default:  ▷ Running after_composer hook
    default:  * Provisioning PHP_CodeSniffer (phpcs), see https://github.com/squizlabs/PHP_CodeSniffer
    default: Using version ^0.7.1 for dealerdirect/phpcodesniffer-composer-installer
    default: Using version ^2.3 for wp-coding-standards/wpcs
    default: Using version ^2.3 for automattic/vipwpcs
    default: Using version ^9.3 for phpcompatibility/php-compatibility
    default: Using version ^1.3 for phpcompatibility/phpcompatibility-paragonie
    default: Using version ^2.1 for phpcompatibility/phpcompatibility-wp
    default: ./composer.json has been updated
    default: Running composer update dealerdirect/phpcodesniffer-composer-installer wp-coding-standards/wpcs automattic/vipwpcs phpcompatibility/php-compatibility phpcompatibility/phpcompatibility-paragonie phpcompatibility/phpcompatibility-wp --with-all-dependencies
    default: Loading composer repositories with package information
    default: Updating dependencies
    default: Nothing to modify in lock file
    default: Installing dependencies from lock file (including require-dev)
    default: Nothing to install, update or remove
    default: Generating autoload files
    default:  * Symlinking phpcs and phcbf into /usr/local/bin
    default:  * Setting WordPress-Core as the default PHPCodesniffer standard
    default: Using config file: /srv/www/phpcs/vendor/squizlabs/php_codesniffer/CodeSniffer.conf
    default: 
    default: Config value "default_standard" updated successfully; old value was "WordPress-Core"
    default:  * The following PHPCS standards are set up:
    default: The installed coding standards are PSR12, PSR1, PSR2, MySource, Squiz, PEAR, Zend, WordPress-VIP-Go, WordPressVIPMinimum, PHPCompatibility, PHPCompatibilityParagonieRandomCompat, PHPCompatibilityParagonieSodiumCompat, PHPCompatibilityWP, VariableAnalysis, WordPress-Core, WordPress-Docs, WordPress-Extra and WordPress
    default:  * PHPCS provisioning has ended
    default:  ✔ Finished after_composer hook in 3s
    default:  * Updating Grunt CLI
    default: npm
    default:  
    default: ERR!
    default:  
    default: code
    default:  EEXIST
    default: npm
    default:  ERR! syscall symlink
    default: npm ERR!
    default:  path ../lib/node_modules/grunt-cli/bin/grunt
    default: npm ERR! dest
    default:  /usr/bin/grunt
    default: npm ERR! errno -17
    default: npm
    default:  ERR! EEXIST: file already exists, symlink '../lib/node_modules/grunt-cli/bin/grunt' -> '/usr/bin/grunt'
    default: npm
    default:  ERR!
    default:  File exists: /usr/bin/grunt
    default: npm ERR! Remove the existing file and try again, or run npm
    default: npm ERR!
    default:  with --force to overwrite files recklessly.
    default: npm
    default:  
    default: ERR!
    default:  A complete log of this run can be found in:
    default: npm ERR!
    default:      /root/.npm/_logs/2021-05-25T18_41_04_062Z-debug.log
    default:  ! The 'main' provisioner ran into problems, the full log is available at '/var/log/provisioners/2021.05.25_18-40-03/provisioner-main.log'. It completed in 61 seconds.
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```
</details>

Specifically:

```
    default:  * PHPCS provisioning has ended
    default:  ✔ Finished after_composer hook in 3s
    default:  * Updating Grunt CLI
    default: npm
    default:  
    default: ERR!
    default:  
    default: code
    default:  EEXIST
    default: npm
    default:  ERR! syscall symlink
    default: npm ERR!
    default:  path ../lib/node_modules/grunt-cli/bin/grunt
    default: npm ERR! dest
    default:  /usr/bin/grunt
    default: npm ERR! errno -17
    default: npm
    default:  ERR! EEXIST: file already exists, symlink '../lib/node_modules/grunt-cli/bin/grunt' -> '/usr/bin/grunt'
    default: npm
    default:  ERR!
    default:  File exists: /usr/bin/grunt
    default: npm ERR! Remove the existing file and try again, or run npm
    default: npm ERR!
    default:  with --force to overwrite files recklessly.
    default: npm
    default:  
    default: ERR!
    default:  A complete log of this run can be found in:
    default: npm ERR!
    default:      /root/.npm/_logs/2021-05-25T18_41_04_062Z-debug.log
    default:  ! The 'main' provisioner ran into problems, the full log is available at '/var/log/provisioners/2021.05.25_18-40-03/provisioner-main.log'. It completed in 61 seconds.
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -41,6 +41,7 @@ permalink: /docs/en-US/changelog/
 * Fixed Parallels mount permissions
 * Fixes for site names containing spaces causing Nginx and TLS issues
 * Warnings that you're missing vagrant plugins no longer show when running vagrant plugin commands
+* Force the installation and update of grunt and grunt-cli so that old grunt is always overwritten when updated
 
 ## 3.6.2 ( 2021 March 17th )
 

--- a/provision/core/grunt/provision.sh
+++ b/provision/core/grunt/provision.sh
@@ -3,7 +3,7 @@
 
 function install_grunt() {
   vvv_info " * Installing Grunt CLI"
-  npm_config_loglevel=error npm install -g grunt grunt-cli --no-optional
+  npm_config_loglevel=error npm install -g grunt grunt-cli --no-optional --force
   npm_config_loglevel=error hack_avoid_gyp_errors & npm install -g grunt-sass --no-optional; touch /tmp/stop_gyp_hack
   npm_config_loglevel=error npm install -g grunt-cssjanus --no-optional
   npm_config_loglevel=error npm install -g grunt-rtlcss --no-optional
@@ -12,7 +12,7 @@ function install_grunt() {
 
 function update_grunt() {
   vvv_info " * Updating Grunt CLI"
-  npm_config_loglevel=error npm update -g grunt grunt-cli --no-optional
+  npm_config_loglevel=error npm update -g grunt grunt-cli --no-optional --force
   npm_config_loglevel=error hack_avoid_gyp_errors & npm update -g grunt-sass; touch /tmp/stop_gyp_hack
   npm_config_loglevel=error npm update -g grunt-cssjanus --no-optional
   npm_config_loglevel=error npm update -g grunt-rtlcss --no-optional

